org-code-javabuilder/lib/src/main/java/org/code/javabuilder/ProjectData.java

```diff
@@ -83,4 +82,14 @@ public void addNewAssetUrl(String filename, String url) {
 
     this.jsonData.getJSONObject(ASSET_URLS_KEY).put(filename, url);
   }
+
+  public UserProjectFiles getValidation() throws UserInitiatedException, InternalServerError {
+    if (!this.jsonData.has(VALIDATION_KEY)) {
+      // return empty file list if there is no validation, as no validation is expected behavior
+      return new UserProjectFiles();
+    }
+    final UserProjectFiles validationFiles =
+        this.projectFileParser.parseFileJson(this.jsonData.getString(VALIDATION_KEY));
+    return validationFiles;
```

Comment:
NIT: would it make sense to just return `this.projectFileParser.parseFileJson(this.jsonData.getString(VALIDATION_KEY));`?